maintainers/maintainer-list.nix

```diff
@@ -5391,4 +5391,9 @@
     github = "minijackson";
     name = "Rémi Nicole";
   };
+  shazow = {
```

Comment:
The last several entries in the maintainer list aren't alphabetic. Should I reorder the whole thing, or are we doing appending now?

Reply:
I'd just order it in a commit before the one adding yourself.

Reply:
I think the ordering of the maintainer list is not related to this PR, so I recommend not worrying about it. I'm worried the reordering will cause a bunch of merge conflicts in new PRs too.